sandbox/grist/schema.py

```diff
@@ -261,7 +261,8 @@ def schema_create_actions():
       make_column("label", "Text"),
       make_column("memo", "Text"),
       make_column("enabled", "Bool"),
-      make_column("watchedColRefList", "RefList:_grist_Tables_column")
+      make_column("watchedColRefList", "RefList:_grist_Tables_column"),
+      make_column("options", "Text"),
```

Comment:
related to https://github.com/gristlabs/grist-core/issues/827

Reply:
I am not sure whether it is a good idea to add this column before the feature is actually implemented, as per the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle.

Reply:
I am agree with this principal. I added it to make it easier after as @paulfitz proposed in https://github.com/gristlabs/grist-core/issues/827

Reply:
OK then :)

Reply:
The principle makes sense in general, but I'm willing to gamble a little here that we'll actually get around to using this column, if the reward is saving a migration.